hoodie-client/src/main/java/com/uber/hoodie/config/HoodieHBaseIndexConfig.java

```diff
@@ -69,6 +76,34 @@
    */
   public static final float DEFAULT_HBASE_QPS_FRACTION = 0.5f;
 
+  /**
+   *  Property to decide if HBASE_QPS_FRACTION_PROP is dynamically calculated based on volume
+   */
+  public static final String HOODIE_INDEX_COMPUTE_QPS_DYNAMICALLY = "hoodie.index.hbase.dynamic_qps";
+  public static final boolean DEFAULT_HOODIE_INDEX_COMPUTE_QPS_DYNAMICALLY = false;
+  /**
+   *  Min and Max for HBASE_QPS_FRACTION_PROP to stabilize skewed volume workloads
+   */
+  public static final String HBASE_MIN_QPS_FRACTION_PROP = "hoodie.index.hbase.min.qps.fraction";
+  public static final String DEFAULT_HBASE_MIN_QPS_FRACTION_PROP = "0.002";
+
+  public static final String HBASE_MAX_QPS_FRACTION_PROP = "hoodie.index.hbase.max.qps.fraction";
+  public static final String DEFAULT_HBASE_MAX_QPS_FRACTION_PROP = "0.06";
+  /**
+   *  Hoodie index desired puts operation time in seconds
+   */
+  public static final String HOODIE_INDEX_DESIRED_PUTS_TIME = "hoodie.index.hbase.desired_puts_time";
```

Comment:
Is this puts time in millis or seconds ? Can you rename this to HOODIE_INDEX_DESIRED_PUTS_TIME_IN_SECS or whatever it is..

Reply:
updated.